webapp/src/selectors.js

```diff
@@ -48,3 +48,4 @@ export const getSiteURL = () => {
 };
 
 export const isRhsVisible = (state) => getPluginState(state).isRhsVisible;
+export const isTeamSidebarHidden = (state) => getPluginState(state).isTeamSidebarHidden;
```

Comment:
Can we have this selector be `isTeamSidebarVisible`? It's fine to have the config and reducer be the `hidden` version, but I think for the selector it makes sense to use `visible`.